qconsolidatedialog.py

```diff
@@ -98,7 +111,7 @@ def accept(self):
                                           QFileInfo(projectFile).fileName())
```

Comment:
With a proper default in the UI I would put `project_name` instead of `QFileInfo(projectFile).fileName()` also here. What's your opinion?